src/countdown-timer/countdown-timer.style.tsx

```diff
@@ -0,0 +1,86 @@
+import styled, { css } from "styled-components";
+import { Color } from "../color";
+import { TextStyleHelper } from "../text";
+import { MediaQuery } from "../media";
+
+// =============================================================================
+// STYLE TYPES
+// =============================================================================
+
+interface CountdownStyleProps {
+    $isFixed: boolean;
+    $opacity: boolean;
+    $warn: boolean;
+    $pinned: boolean;
+    $top: number;
+    $left: number;
+}
+
+// =============================================================================
+// STYLING
+// =============================================================================
+
+export const Wrapper = styled.div`
+    width: 100%;
+`;
+
+export const Countdown = styled.div<CountdownStyleProps>`
+    ${TextStyleHelper.getTextStyle("H4", "semibold")}
+    display: inline-flex;
```

Comment:
is there a specific reason to use `inline-flex` over `flex`?

Reply:
nope actually.